dubbo-common/pom.xml

```diff
@@ -77,5 +77,9 @@
             <groupId>de.ruedigermoeller</groupId>
             <artifactId>fst</artifactId>
         </dependency>
+        <dependency>
```

Comment:
We should not introduce this dependency in the common. If only GenericFilter relies on it, pls. move it to "dubbo-rpc-api" module.